index.html

```diff
@@ -8,55 +8,68 @@
 </head>
 <body>
-
     <header>
       <nav>
+        <div class="left">
         <a class="navbar-logo" href="#">
             My Logo
         </a>
-        <label class="navbar-hamburger" for="toggle">
+        </div>
+
+    <div class="right">   
+            <label class="navbar-hamburger" for="toggle">
             <span class="bar"></span>
             <span class="bar"></span>
             <span class="bar"></span>
-        </label>
-      </nav>
+            </label>  
+        <div id="main_links"> 
+        <a class="main-links">Portfolio</a>
+        <a class="main-links">About</a>
+        <a class="main-links">Contact</a> 
+        </div>
+    </div>
+      </nav> 
     </header>
 
+   
 
-    <section class="about" id="about">
 
-    <h1 class="title">
-            I’m Miguel Cuello.
+    <section class="main" id="about">
+    <div id="one">
+        <h1 class="title">
+            I’m Miguel.
             Glad to see you!
     </h1>
-        <p class="text">
+        <p class="text" id="main_text">
             I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
-        </p>
-
-    <p id="color">LET’S CONNECT</p>
-
-    <ul class="social">
-        <li><a href="#"><img src="images/twitter.svg" id="twitter" class="icon"></a></li>
-        <li><a href="#"><img src="images/linkedin.svg"id="linkedin" class="icon"></a></li>
-        <li><a href="#"><img src="images/hand.svg" id="hand" class="icon"></a></li>
-        <li><a href="#"><img src="images/git.svg" id="github" class="icon"></a></li>
-        <li><a href="#"><img src="images/M.svg" id="m" class="icon"></a></li>
-    </ul>
-
-    </section>
+        </p> 
+    <div class="lets-connect" >
+        <p id="color">LET’S CONNECT</p>
+
+    <div id="social">
+       <a href="#"><img src="images/twitter.svg" id="twitter" class="icon"></a>
+       <a href="#"><img src="images/linkedin.svg"id="linkedin" class="icon"></a>
+       <a href="#"><img src="images/hand.svg" id="hand" class="icon"></a>
+       <a href="#"><img src="images/git.svg" id="github" class="icon"></a>
+       <a href="#"><img src="images/M.svg" id="m" class="icon"></a>
+    </ul></div>
+    
+    </div>
+</section>
 
-    <section  class="card">
+<section id="cards" >
 
-        <img src="images/portfolio.png" alt="a_screentshot" class="pic">
+    <div class="card" id="card_one">
+        <img src="images/portfolio.png" alt="a_screentshot" class="tonic screen-pic">
 
-        <div class="text-card">
+          <div class="text-card">
             <h2 class="title cards">Tonic</h2>
 
-        <img src="images/fame2.png" alt="frame" class="frame">
+              <img src="images/fame2.png" alt="frame" class="frame">
 
-        <p class="text">
-            A daily selection of privately personalized reads; no accounts or sign-ups required.
```

Comment:
- [ ] Kindly fix the following.

- Your design
<img width="411" alt="image" src="https://github.com/migcm06/miguel-portfolio/assets/41163571/0d50e5d6-64ad-4cd8-a9e6-c49b00d1f476">

- Figma design
<img width="409" alt="image" src="https://github.com/migcm06/miguel-portfolio/assets/41163571/bb2b35b7-4d4a-46d0-8a4c-464b8e8aa077">


1. Kindly reduce the top margin to match the design.
2. Kindly make sure that the padding of the `html` `css` and javascript match the Figma design, by making the padding rhyme, currently it has more top and bottom padding than left and right.
3. Kindly reduce the padding of button, to match the original design.